src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return days * 40 - 20;
```

Comment:
Replace the literal `20` with a constant that represents the discount for a medium term rental. Use a descriptive name like `MEDIUM_TERM_DISCOUNT`.